app/views/shared/check_answers/_applied_previously.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= govuk_summary_list(card: { title: t(".heading") },
```

Comment:
Shouldn't this be `govuk_summary_card(title: t(".heading"), heading_level: 3)`
Or is that the `old` format?

Reply:
Nope, agreed, should be `govuk_summary_card`